pywbem_mock/_defaultinstanceprovider.py

```diff
@@ -821,8 +793,16 @@ class MethodProvider(BaseProvider):
 
     User providers are defined by creating a subclass of this class and
     defining an InvokeMethod based on the method in this class.
+
+    Attributes:
+
+      provider_type (:term:`string`):
+        Keyword defining the type of request the provider will service.
+        The type for this class is 'method'
```

Comment:
This way of describing the `provider_type` attribute causes Sphinx not to create proper links to the attribute. I verified that when changing this into line comments, it works fine:

```
    #:  provider_type (:term:`string`):
    #:    Keyword defining the type of request the provider will service.
    #:    The type for this class is predefined as 'instance'
    provider_type = 'method'
```

Reply:
DONE